dist/s_string.ok

```diff
@@ -694,7 +694,6 @@ dmb
 dmsg
 drealloc
 dryrun
-ds
```

Comment:
Question not about this change: How do you know what strings to delete? ./s_all tells you what strings to add. Does it also tell what strings to delete?

Reply:
There is a `-r` flag to `s_string` that will build a replacement list of words. That gets periodically run to trim entries no longer needed as code gets removed.